stacks/awx-operator/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add awx-operator https://ansible.github.io/awx-operator/
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="awx-operator"
+CHART="awx-operator"
+CHART_VERSION="1.1.1"
```

Comment:
Looks like this needs to be updated to 1.1.2 for this to work.  The current version is not found.

Reply:
Surely, might be related to [this](https://github.com/ansible/awx-operator/issues/1136).